build/windows/gh.wxs

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?define UpgradeCode=8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4?>
```

Comment:
Yes. It's a new product installing to a new directory. It will still upgrade (remove) the old one. In my many years of experience with Windows Installer (since it was "Darwin"), I've found a clean break better in cases like this.